crates/orderbook/src/fee.rs

```diff
@@ -342,15 +355,13 @@ impl MinFeeCalculating for MinFeeCalculator {
         app_data: AppId,
         subsidized_fee: U256,
         user: H160,
-    ) -> Result<FeeParameters, ()> {
+    ) -> Result<FeeParameters, GetUnsubsidizedMinFeeError> {
         let cow_factor = self.cow_subsidy.cow_subsidy_factor(user);
         let past_fee = self
             .measurements
             .find_measurement_including_larger_amount(fee_data, (self.now)());
         // TODO: make this function return a proper error distinguishing errors from fee too low;
-        let (cow_factor, past_fee) = futures::future::try_join(cow_factor, past_fee)
-            .await
-            .map_err(|_| ())?;
+        let (cow_factor, past_fee) = futures::future::try_join(cow_factor, past_fee).await?;
```

Comment:
Is that `TODO` still relevant? It looks like the error propagation is now correct, no?